README.md

```diff
@@ -0,0 +1,93 @@
+Prometheus Load generator for GDS RE Observe
+============================================================================
+
+This is a python scripted composed using locust as the source of load generation. The tool queries a remote prometheus in order to generate a list of queries base on metrics and a time interval.
```

Comment:
I think referring to the "remote prometheus" instead by something like "prometheus under test" might be clearer. Similarly for the below sentence - "These queries are then run on the againest the prometheus server." we could be more explicit about if that is the local Prometheus or the Prometheus under test.

Also I think we need something summarising that we are and explaining why we are also running a prometheus container (in addition to the one that is under test). It's not immediately clear to me when reading this.

It would also be nice to link "locust" to the website for the tool please. 

There are also a few small spelling/grammar points in this intro which would be nice to fix
- "python scripted composed using locust" could maybe be "python script which uses Locust"
- "list of queries base on metrics" should be 'based' instead of 'base' I think
- 'queries' instead of 'quries'
- 'static' instead of 'staic'
- 'against' instead of 'againest'

Happy to help with this intro/pair on this so do let me know if that would be helpful :)